init.sh

```diff
@@ -24,8 +37,11 @@ make install
 evmosd config keyring-backend $KEYRING
 evmosd config chain-id $CHAINID
 
-# if $KEY exists it should be deleted
-evmosd keys add $KEY --keyring-backend $KEYRING --algo $KEYALGO
+# If keys exist they should be deleted
+for KEY in "${KEYS[@]}"
+do
+  evmosd keys add $KEY --keyring-backend $KEYRING --algo $KEYALGO
+done
 
 # Set moniker and chain-id for Evmos (Moniker can be anything, chain-id must be an integer)
 evmosd init $MONIKER --chain-id $CHAINID
```

Comment:
```suggestion
evmosd init $MONIKER --chain-id $CHAINID -o
```

Reply:
Added, along with the specification of `--home $homedir` for all of the commands